packages/components/src/slot-fill/test/observable-map.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import { render, screen, act } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import {
+	observableMap,
+	useObservableValue,
+} from '../bubbles-virtually/observable-map';
+
+describe( 'ObservableMap', () => {
+	test( 'should observe individual values', () => {
+		const map = observableMap();
+		let notifsFromA = 0;
+		let notifsFromB = 0;
+
+		const unsubA = map.subscribe( 'a', () => {
+			notifsFromA++;
+		} );
+		const unsubB = map.subscribe( 'b', () => {
+			notifsFromB++;
+		} );
+
+		// check that setting `a` doesn't notify the `b` listener
+		map.set( 'a', 1 );
+		expect( notifsFromA ).toBe( 1 );
+		expect( notifsFromB ).toBe( 0 );
+
+		// check that setting `b` doesn't notify the `a` listener
+		map.set( 'b', 2 );
+		expect( notifsFromA ).toBe( 1 );
+		expect( notifsFromB ).toBe( 1 );
+
+		// check that `delete` triggers notifications, too
+		map.delete( 'a' );
+		expect( notifsFromA ).toBe( 2 );
+		expect( notifsFromB ).toBe( 1 );
+
+		// check that the subscription survived the `delete`
+		map.set( 'a', 2 );
+		expect( notifsFromA ).toBe( 3 );
+		expect( notifsFromB ).toBe( 1 );
+
+		// check that unsubscription really works.
+		unsubA();
+		unsubB();
+		map.set( 'a', 3 );
+		expect( notifsFromA ).toBe( 3 );
+		expect( notifsFromB ).toBe( 1 );
```

Comment:
Minor: Should those subscription assertions be in separate tests? This particular one tests that after unsubscribing individual values are no longer observed, which is exactly the opposite of what the test claims to do 🤔